tests_e2e/lisa/runbook/azure.yml

```diff
@@ -0,0 +1,82 @@
+name: azure
```

Comment:
Currently, the interface with LISA is under the 'lisa' subdirectory (it will be moved one level up in a future PR). This YML file is the entry point to LISA.